### PR TITLE
support get statistics

### PR DESCRIPTION
**Pull request by siddontang**
Use this function to see the RocksDB statistics. 

@ngaut @BusyJay @zhangjinpeng1987

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
LGTM

File: src/rocksdb_options.rs [hunk 1]
```diff
@@ -515,6 +515,22 @@ impl Options {
...
+            let value = rocksdb_ffi::rocksdb_options_statistics_get_string(self.inner);
+
+
+            if value.is_null() {
+                return None;
+            }
+
+            // Must valid UTF-8 format.
+            let s = CStr::from_ptr(value).to_str().unwrap().to_owned();
+            libc::free(value as *mut c_void);
```
Comment (BusyJay) [inline src/rocksdb_options.rs]: Does it need to be freed?

Comment (siddontang) [inline src/rocksdb_options.rs]: yes, it returns a malloc string.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**